Gemfile

```diff
@@ -37,6 +37,8 @@ gem "countries"
 
 gem "hashid-rails" # Non-sequential IDs
 
+gem "premailer-rails" # Inline CSS for emails
```

Comment:
Should we move this under sprockets?

Reply:
[`a50b043` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/a50b04378d15baa9a835d3920482a2ef8b162deb)